### PR TITLE
Add HackerOne

### PR DESCRIPTION
**Pull request by jobertabma**

### CODE REVIEW

**Review by Carlgo11 — DISMISSED** [review]
Here's a better logo. It's much smaller and transparent. Taken directly from their website.
logo

File: _data/security.yml [hunk 1]
```diff
@@ -127,6 +127,13 @@ websites:
       twitter: gogetssl
       facebook: gogetssl
 
+    - name: HackerOne
+      url: https://hackerone.com
+      img: hackerone.png
+      tfa: Yes
+      software: Yes
+      doc: https://docs.hackerone.com/hackers/two-factor-authentication.html
+
```
Comment (reedloden) [inline _data/security.yml:136]: Maybe also add `twitter` and `facebook`, too?

Comment (Carlgo11) [reply _data/security.yml:136]: @reedloden not if `tfa` is set to Yes. See contributing.md

**Comment by reedloden** [comment]
Oh, oops. My bad. :-)

**Review by brycx — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**